logstash-integration-aws.gemspec

```diff
@@ -0,0 +1,46 @@
+Gem::Specification.new do |s|
+  s.name            = "logstash-integration-aws"
+  s.version         = "0.1.0"
+  s.licenses        = ["Apache-2.0"]
+  s.summary         = "Collection of Logstash plugins that integrate with AWS"
+  s.description     = "This gem is a Logstash plugin required to be installed on top of the Logstash core pipeline using $LS_HOME/bin/logstash-plugin install gemname. This gem is not a stand-alone program"
+  s.authors         = ["Elastic"]
+  s.email           = "info@elastic.co"
+  s.homepage        = "http://www.elastic.co/guide/en/logstash/current/index.html"
+  s.metadata        = {
+    "logstash_plugin" => "true",
+    "logstash_group" => "integration",
+    "integration_plugins" => %w(
+      logstash-codec-cloudfront
+      logstash-codec-cloudtrail
+      logstash-input-cloudwatch
+      logstash-input-s3
+      logstash-input-sqs
+      logstash-mixin-aws
```

Comment:
This list is used for three purposes:
1. show up in the list during `bin/logstash-plugin list`
2. remove any plugin from that list after installing the integration (cleanup)
3. prevent manual installation of plugins from that list

Point 2 isn't valid as the aws-sdk conflict won't let the installation proceed to this point.
Point 1 suggests we should remove the mixin from this list as it's not helpful to see the mixin in "plugin list"
Point 3 is the only reason to keep it, so that `bin/logstash-plugin install logstash-mixin-aws` fails if the integration is installed.